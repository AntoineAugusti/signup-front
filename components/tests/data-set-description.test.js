import ReactTestRenderer from 'react-test-renderer'
import DataSetDescription from '../data-set-description'
import RESOURCE_PROVIDERS from '../../mock/data/resource-providers'

const RESOURCE_PROVIDER = RESOURCE_PROVIDERS[0]

describe('components | DataSetDescription', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<DataSetDescription dataset={RESOURCE_PROVIDER.scopes[0]} provider={RESOURCE_PROVIDER.short_name} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
