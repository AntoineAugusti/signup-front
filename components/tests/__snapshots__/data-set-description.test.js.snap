// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | DataSetDescription render should match snapshot 1`] = `
<ul>
  <li>
    <h3>
      Avis d‘imposition
    </h3>
    <p>
      Contient le revenu fiscal de référence, le nombre de part et les déclarants
    </p>
    <h3>
      Qui utilise ces données ?
    </h3>
    <div>
      <p>
        Mes aides
      </p>
      <img
        alt="Mes aides logo"
        src="./static/images/services-providers/mes-aides-logo.png"
      />
    </div>
    <div>
      <p>
        Anah
      </p>
      <img
        alt="Anah logo"
        src="./static/images/services-providers/anah-logo.png"
      />
    </div>
    <a
      className="button"
      href="https://particulier.api.gouv.fr/tech/#r-cup-rer-les-donn-es-fiscales-d-39-un-citoyen"
      onClick={[Function]}
    >
      Essayez-moi !
    </a>
    <button
      className="button"
      id="submit"
      name="add-to-AddToSelectionButton"
      type="submit"
    >
      Ajouter à ma sélection
    </button>
  </li>
</ul>
`;
